### PR TITLE
Major version. 1.0 Wordpress + Woocommerce PART 2

### PR DESCRIPTION
**Pull request by eduardogomez97**
Required by: https://github.com/orgs/doofinder/projects/19/views/6?pane=issue&itemId=24897210

This PR includes everything you need to have the wizard-setup adapted to wordpress.

### CODE REVIEW

File: doofinder/includes/class-setup-wizard.php [hunk 1]
```diff
@@ -120,54 +396,274 @@ public static function get_step() {
...
+
+		$message_intro = __('<strong>Welcome to Doofinder</strong>', 'wordpress-doofinder');
+
+		$message = __(' &#8211; Run setup wizard to finish installation.', 'wordpress-doofinder');
+
+		if (Settings::is_configuration_complete()) {
+			$message = __(' &#8211; Looks like Doofinder is already set up. You can review the configuration in the settings or run the setup wizard.', 'wordpress-doofinder');
+		}
+
+		ob_start();
```
Comment (pedromcp90) [inline doofinder/includes/class-setup-wizard.php:549]:
```suggestion
		ob_start();
		?>
		<div class="notice notice-success is-dismissible">
			<div id="message" class="wordpress-message doofinder-notice-setup-wizard">
				<figure class="logo" style="width:5rem;height:auto;float:left;margin:.5em 0;margin-right:0.75rem;">
					<img src="<?php echo Doofinder_For_WordPress::plugin_url() . 'assets/svg/imagotipo1.svg'; ?>" />
				</figure>
				<h3><?php echo $message_intro;?></h3>
				<p><?php echo $message;?></p>
				<p class="submit">
					<a href="<?php echo self::get_url(); ?>" class="button-primary button-setup-wizard"><?php _e('Setup Wizard', 'wordpress-doofinder'); ?></a>
					<?php if ($settings) : ?>
						&nbsp;<a class="button-secondary button-settings" href="<?php echo Settings::get_url(); ?>"><?php _e('Settings', 'wordpress-doofinder'); ?></a>
					<?php endif; ?>
				</p>
			</div>
		</div>
		<?php
		$html = ob_get_clean();
		return $html;
```

File: doofinder/views/wizard.php [hunk 2]
```diff
@@ -2,24 +2,58 @@
...
+<!DOCTYPE html>
+<html <?php language_attributes(); ?>>
 
-<html>
 <head>
-    <title><?php printf( __( '%s - Doofinder Setup Wizard', 'doofinder_for_wp' ), get_bloginfo( 'name' ) ); ?></title>
-
+    <meta name="viewport" content="width=device-width" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title><?php printf(__('%s - Doofinder Setup Wizard', 'woocommerce-doofinder'), get_bloginfo('name')); ?></title>
```
Comment (pedromcp90) [inline doofinder/views/wizard.php:21]:
```suggestion
    <title><?php printf(__('%s - Doofinder Setup Wizard', 'wordpress-doofinder'), get_bloginfo('name')); ?></title>

```

**Review by pedromcp90 — CHANGES_REQUESTED** [review]

**Comment by pedromcp90** [comment]
Please add this missing file that is needed to be shown in the Wizard setup notice.
`/assets/svg/imagotipo1.svg`

**Comment by pedromcp90** [comment]
The create-store endpoint used by the Wizard Setup is failing.
You should check whether it is a payload issue or something wrong in doofapi or doomanager.

**Review by pedromcp90 — APPROVED** [review]

**Comment by eduardogomez97** [comment]
This has already been merged into the refactor branch

**Pull request closed without merging**